src/commands.py

```diff
@@ -75,6 +75,27 @@ def difficulties(ep):
     tabulate(sorted_diffi, headers="keys", tablefmt="orgtbl", stralign="center")))
 
 
+network_sort_keys = ["block", "member", "diffi", "uid"]
+
+
+def set_network_sort_keys(some_keys):
+    global network_sort_keys
```

Comment:
Why do you need to get this variable here?
It's used bellow and should works without doing this?